pretix_mollie/payment.py

```diff
@@ -116,13 +122,24 @@ def settings_form_fields(self):
             else:
                 return {}
         else:
+            api_key_kw = {}
+            if self.settings.api_key:
+                api_key_kw['required'] = False
+                # We can show the first 5 characters of the key, as it indicates test or live key.
+                api_key_kw['help_text'] = (_("A secret key has currently been configured: %s. The remainder of the "
+                                             "secret key has been hidden for security purposes.") %
+                                           (self.settings.api_key[:5] + "*" * len(self.settings.api_key[5:])))
```

Comment:
Hmm, maybe it looks nicer if we set the shorter value as a placeholder instead of the help text? I think that's clear enough and requires less text. Nobody reads text :)